content/source/docs/enterprise/upgrade/differences.html.md

```diff
@@ -24,7 +24,7 @@ The new Terraform Enterprise does not support the `terraform push` command, whic
 
 This does not mean we're dropping support for a CLI-based workflow — we want push to be better, and the integrated `push` command was holding us back because we built it on an outdated understanding of how Terraform Enterprise should work. Users of `terraform push` have a few options:
 
-- Use [the add-on Terraform Enterprise command line tools](https://github.com/hashicorp/tfe-cli). These are designed as a one-to-one replacement for `terraform push`, with a modified syntax and some new features.
+- Use the [community supported Terraform CLI helper](https://github.com/hashicorp-community/tf-helper). This was designed as a one-to-one replacement for `terraform push`, with a modified syntax and some new features.
```

Comment:
@fprimex Does this work for the new tfh? 

@mikenomitch I'm wondering if we ought to consider just removing this page, as I think it's mostly out of date now. This was useful during the time that people were transitioning to the new workflow/UI.

Reply:
`tfh pushconfig` and `tfh pushvars` are the same as in `tfe-cli`, yep.

Reply:
@armchairlinguist - yeah I wondered about removing the page too. Who would be the person/people to make that call? (hoping its you 😄 )

Reply:
FYI, I spoke with the rest of the team and they'll handle this in a different PR/ticket.